conan/tools/meson/toolchain.py

```diff
@@ -464,9 +466,9 @@ def _sanitize_format(v):
             # https://mesonbuild.com/Builtin-options.html#directories
             # https://mesonbuild.com/Machine-files.html#binaries
             # https://mesonbuild.com/Reference-tables.html#compiler-and-linker-selection-variables
-            "c": _sanitize_format(self.c),
-            "cpp": _sanitize_format(self.cpp),
-            "ld": _sanitize_format(self.ld),
+            "c": to_meson_value(self.c),
```

Comment:
why is it needed to use `to_meson_value` here?

Reply:
Because the `sanitize` method was adding the quotes needed for Meson and we are no longer using it in some cases.

Reply:
ok, I see now